### ISSUE TITLE
Grows

### ISSUE THREAD

**Issue by MNieginski** [MNieginski/GiphySearch#2]
I feel like the code is a little chaotic/messy, so it can definitely be cleaned up a little, 
overall design can use an update,
maybe some some more features

**Comment by krabecb** [comment]
Yes, I agree! Definitely find some time to identify those additional features! 💯
